includes/cenchat-comments.php

```diff
@@ -0,0 +1,10 @@
+<?php
+class Cenchat_Comments {
+    public function apply_template( $comment_template ) {
+        return dirname( dirname( __FILE__ ) ) . '/public/partials/template.php';
```

Comment:
Use [`plugin_dir_path`](https://developer.wordpress.org/reference/functions/plugin_dir_path/) instead of nested `dirname()`.